docs/.eleventy.js

```diff
@@ -71,8 +71,26 @@ module.exports = function(eleventyConfig) {
      */
     function slugify(text) {
         return slug(text.replace(/[<>()[\]{}]/gu, ""))
-        // eslint-disable-next-line no-control-regex -- used regex from https://github.com/eslint/archive-website/blob/master/_11ty/plugins/markdown-plugins.js#L37
-            .replace(/[^\u{00}-\u{FF}]/gu, "");
+        /* eslint-disable no-control-regex, require-unicode-regexp -- used regex to replace */
+
+        // Remove control characters
+            .replace(/[\u0000-\u001F]/g, "")
+
+        // replace special characters
+            .replace(/[\s~`!@#$%^&*()\-_+=[\]{}|\\;:"'<>,.?/]+/g, "-")
+
+        // remove continuos separators
+
+            .replace(/-{2,}/g, "-")
+
+        // remove prefixing and trailing separtors
+            .replace(/^-+|-+$/g, "")
+
+        /* eslint-enable no-control-regex, require-unicode-regexp -- end */
+
+        // lowercase
+            .toLowerCase();
```

Comment:
Would a simple `return slug(text);` work well for zh-hans docs? I just tried it here and didn't see any diff in `site/` so it looks like we could easily make that change.